_posts/2024-05-28-edb-and-kasten.markdown

```diff
@@ -0,0 +1,295 @@
+---
+layout: post
+title: Protect your postgres EDB database with Kasten
+description: Kasten and EDB have worked on a partnership so that you can protect and move your EDB database using Kasten. With this partnership we get best of both words, a transaction consistent backup, based on storage snapshot and entirely managed by Kasten ! This will let you execute disaster recovery and migration scenario of enterprise database in just few clicks.
+date: 2024-05-28 13:48:00 +0000
+author: michaelcourcy
+image: '/images/posts/2024-05-28-edb-and-kasten/kasten-edb-2.jpg'
+image_caption: 'EDB and Kasten partnership'
+tags: [Disaster Recovery, Migration, EDB, Kasten, RTO, Partnership, Managed Database]
+featured:
+---
+
+# Why database on Kubernetes ?
+
+ Having your database in Kubernetes has several advantages :   
+- **Performance** : you colocalize the data and the application
+- **Security** : All stay within the kubernetes network
+- **Ease of deployment** : `kubectl apply -f ...` and you're good to go. 
+- **Automation** : You define your desired state, the operator make it happens.
+- **Self Healing** : As it benefit to any workload in Kubernetes.
```

Comment:
As it benefit to any workload in Kubernetes.  --> Benefits for all workloads in Kubernetes.